### PR TITLE
Dns txt records

### PR DESCRIPTION
**Pull request by ndemoor**

### CODE REVIEW

File: lib/dnsTxtRecordParser.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const definitions = require('./definitions');
+
+async function analyze (data) {
+  let technologies = [];
```
Comment (Mickael-van-der-Beek) [inline lib/dnsTxtRecordParser.js]: Can be `const` since no reassignment happens.

File: lib/dnsTxtRecordParser.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+  if(data.dnsTxtRecords) {
+    data.dnsTxtRecords.forEach(record => {
+      Object.keys(definitions.apps).forEach(appKey => {
+        var app = definitions.apps[appKey];
+        if(app.dnsTxtRecord) {
+          if(record[0].match(app.dnsTxtRecord)) {
```
Comment (Mickael-van-der-Beek) [inline lib/dnsTxtRecordParser.js]: The two if conditions can be combined to limit indentation / shorten.

File: lib/dnsTxtRecordParser.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+const definitions = require('./definitions');
+
+async function analyze (data) {
+  let technologies = [];
+
+  if (data.dnsTxtRecords) {
+    data.dnsTxtRecords.forEach(record => {
+      Object.keys(definitions.apps).forEach(appKey => {
+        var app = definitions.apps[appKey];
+        if (app.dnsTxtRecord) {
```
Comment (Mickael-van-der-Beek) [inline lib/dnsTxtRecordParser.js]: The two if conditions can be combined to limit indentation / shorten.

File: lib/dnsTxtRecordParser.js (hunk 2 above)
Comment (ndemoor) [reply lib/dnsTxtRecordParser.js]: All in fir less lines, But how about readability tho

**Comment by ndemoor** [comment]
@Mickael-van-der-Beek both changes done

File: test/analyze.spec.js [hunk 4]
```diff
@@ -80,6 +80,7 @@ describe('should analyze website elements properly', function () {
     };
 
     const technologies = await wappalyzer.analyze({ js });
+    console.log(technologies);
```
Comment (Janpot) [inline test/analyze.spec.js]: console.log

Comment (ndemoor) [reply test/analyze.spec.js]: merde 🤦‍♂️

**Pull request merged**